### PR TITLE
Made Save Files option for exported GIF

### PR DESCRIPTION
**Pull request by Yusma247**
Save Files Option made

### CODE REVIEW

File: src-tauri/src/editor/mod.rs [hunk 1]
```diff
@@ -22,7 +25,7 @@ pub fn init_editor(app: &AppHandle, video_file: String, size: (u32, u32)) {
     let preview_height_adjusted = EDITOR_WIDTH * height / width;
     let editor_win_height = preview_height_adjusted + TOOLS_HEIGHT;
 
-    let mut editor_win =
+    let editor_win =
```
Comment (RohanPunjani) [inline src-tauri/src/editor/mod.rs:28]: Keep this variable as mutable

File: src-tauri/src/editor/mod.rs [hunk 2]
```diff
@@ -97,16 +100,24 @@ pub async fn export_gif(options: ExportOptions, app_handle: AppHandle) {
 
     let gif_encoder = Arc::new(gif_encoder);
 
-    let gif_name = chrono::Local::now()
+    let _gif_name = chrono::Local::now()
```
Comment (RohanPunjani) [inline src-tauri/src/editor/mod.rs:103]: Can you try using this as a default gif file name, user can edit it too, but if he just wants to quickly save it... A predefined name would be better

**Review by RohanPunjani — CHANGES_REQUESTED** [review]

File: src-tauri/src/editor/mod.rs (hunk 2 above)
Comment (Yusma247) [reply src-tauri/src/editor/mod.rs:103]: Ok.. I've added an option to set a default filename based on the current date and time

File: src-tauri/src/editor/mod.rs (hunk 1 above)
Comment (Yusma247) [reply src-tauri/src/editor/mod.rs]: Done

**Pull request merged**